src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  padding: 0;
+  font-family: Avenir, serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
+}
+
+.search-big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-label::after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search-big-label::after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-small-label::after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
```

Comment:
There are styles that repeat. Combine everything that repeats into a common class. Fix all cases